'transport is closing' in object hash CLI command execution

NeoFS v0.19.0 (current dev-env)
'transport is closing' error occurs on attempt to execute object hash command:
```
$ neofs-cli --rpc-endpoint s01.neofs.devenv:8080 --key Ky6BmCo2NEJMw1ucB6CEbM1eSUem2QKcf2duxzTMVM2Sw4r2qKH7 object hash --cid HPRf4LK5D8AwiYUNVvBD9s5Y9y676EFHq2DaC1C9BpUX --oid D2VnZkKwNvx9JbozUpD6DoPMprWxRpjpwhSsYzoggrHP --range 0:10
Error: sending the request failed: rpc error: code = Unavailable desc = transport is closing
Usage:
```

The same object range command has been executed without issues:
```
$ neofs-cli --rpc-endpoint s01.neofs.devenv:8080 --key Ky6BmCo2NEJMw1ucB6CEbM1eSUem2QKcf2duxzTMVM2Sw4r2qKH7 object range --cid HPRf4LK5D8AwiYUNVvBD9s5Y9y676EFHq2DaC1C9BpUX --oid D2VnZkKwNvx9JbozUpD6DoPMprWxRpjpwhSsYzoggrHP --range 0:10
��M�H��Ђ�
```

The error is caused by panic on server-side:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xcd2f89]

goroutine 38 [running]:
github.com/nspcc-dev/neofs-node/pkg/services/object/get.(*clientWrapper).getObject(0xc00003b4b0, 0xc00027b810, 0xc00058e030, 0x111ab20, 0xc0006f6360)
	github.com/nspcc-dev/neofs-node/pkg/services/object/get/util.go:84 +0x639
github.com/nspcc-dev/neofs-node/pkg/services/object/get.(*execCtx).processNode(0xc00027b810, 0x1129e60, 0xc0003ff700, 0xc000215600, 0xc0002155f0)
	github.com/nspcc-dev/neofs-node/pkg/services/object/get/remote.go:23 +0x23e
github.com/nspcc-dev/neofs-node/pkg/services/object/get.(*execCtx).processCurrentEpoch(0xc00027b810, 0xfada00)
	github.com/nspcc-dev/neofs-node/pkg/services/object/get/container.go:81 +0x2ac
github.com/nspcc-dev/neofs-node/pkg/services/object/get.(*execCtx).executeOnContainer(0xc00027b810)
	github.com/nspcc-dev/neofs-node/pkg/services/object/get/container.go:28 +0x1a5
github.com/nspcc-dev/neofs-node/pkg/services/object/get.(*execCtx).analyzeStatus(0xc00027b810, 0xf9c601)
	github.com/nspcc-dev/neofs-node/pkg/services/object/get/get.go:108 +0x1c0
github.com/nspcc-dev/neofs-node/pkg/services/object/get.(*execCtx).execute(0xc00027b810)
	github.com/nspcc-dev/neofs-node/pkg/services/object/get/get.go:87 +0x79
github.com/nspcc-dev/neofs-node/pkg/services/object/get.(*Service).get(0xc00000e4b0, 0x1129f20, 0xc00058e030, 0x111ab20, 0xc0006f6360, 0xc000538aa0, 0xc000121080, 0x0, 0x0, 0x0, ...)
	github.com/nspcc-dev/neofs-node/pkg/services/object/get/get.go:76 +0x11b
github.com/nspcc-dev/neofs-node/pkg/services/object/get.(*Service).GetRange(0xc00000e4b0, 0x1129f20, 0xc00058e030, 0x111ab20, 0xc0006f6360, 0xc000538aa0, 0xc000121080, 0x0, 0x0, 0x0, ...)
	github.com/nspcc-dev/neofs-node/pkg/services/object/get/get.go:18 +0xd7
github.com/nspcc-dev/neofs-node/pkg/services/object/get.(*Service).GetRangeHash(0xc00000e4b0, 0x1129f20, 0xc00058e030, 0x0, 0x0, 0xc000538aa0, 0xc000121080, 0x0, 0x0, 0x0, ...)
	github.com/nspcc-dev/neofs-node/pkg/services/object/get/get.go:40 +0x2dc
github.com/nspcc-dev/neofs-node/pkg/services/object/get/v2.(*Service).GetRangeHash(0xc00000e4b8, 0x1129f20, 0xc00058e030, 0xc00044bbe0, 0xfd3a48, 0xdb2ba0, 0xc0004f3120)
	github.com/nspcc-dev/neofs-node/pkg/services/object/get/v2/service.go:86 +0xf4
```

--- a/pkg/services/object/get/exec.go
+++ b/pkg/services/object/get/exec.go
@@ -38,7 +38,7 @@ type execCtx struct {
 
 	curOff uint64
 
-	head bool
+	head, hash bool
 
 	curProcEpoch uint64
 
@@ -68,6 +68,12 @@ func headOnly() execOption {
 func withPayloadRange(r *objectSDK.Range) execOption {
 	return func(c *execCtx) {
 		c.prm.rng = r
+	}
+}
+
+func hashOnly() execOption {
+	return func(c *execCtx) {
+		c.hash = true
 	}
 }
 
@@ -144,6 +150,10 @@ func (exec *execCtx) ctxRange() *objectSDK.Range {
 
 func (exec *execCtx) headOnly() bool {
 	return exec.head
+}
+
+func (exec *execCtx) hashOnly() bool {
+	return exec.hash
 }
 
 func (exec *execCtx) netmapEpoch() uint64 {

--- a/pkg/services/object/get/get.go
+++ b/pkg/services/object/get/get.go
@@ -15,7 +15,11 @@ func (s *Service) Get(ctx context.Context, prm Prm) error {
 
 // GetRange serves a request to get an object by address, and returns Streamer instance.
 func (s *Service) GetRange(ctx context.Context, prm RangePrm) error {
-	return s.get(ctx, prm.commonPrm, withPayloadRange(prm.rng)).err
+	return s.getRange(ctx, prm)
+}
+
+func (s *Service) getRange(ctx context.Context, prm RangePrm, opts ...execOption) error {
+	return s.get(ctx, prm.commonPrm, append(opts, withPayloadRange(prm.rng))...).err
 }
 
 func (s *Service) GetRangeHash(ctx context.Context, prm RangeHashPrm) (*RangeHashRes, error) {
@@ -37,7 +41,7 @@ func (s *Service) GetRangeHash(ctx context.Context, prm RangeHashPrm) (*RangeHas
 			hash: util.NewSaltingWriter(h, prm.salt),
 		})
 
-		if err := s.GetRange(ctx, rngPrm); err != nil {
+		if err := s.getRange(ctx, rngPrm, hashOnly()); err != nil {
 			return nil, err
 		}
 

--- a/pkg/services/object/get/util.go
+++ b/pkg/services/object/get/util.go
@@ -80,7 +80,7 @@ func (c *clientCacheWrapper) get(addr string) (getClient, error) {
 }
 
 func (c *clientWrapper) getObject(exec *execCtx) (*objectSDK.Object, error) {
-	if !exec.assembling {
+	if !exec.assembling && !exec.hashOnly() {
 		return exec.prm.forwarder(c.client)
 	}
 

